Emby.Server.Implementations/Data/SqliteItemRepository.cs

```diff
@@ -638,7 +624,7 @@ private void SaveItemsInTransaction(IDatabaseConnection db, IEnumerable<(BaseIte
                 {
                     if (requiresReset)
                     {
-                        saveItemStatement.Reset();
+                        // TODO saveItemStatement.Parameters.Clear();
```

Comment:
Does this TODO have a known fix?

Reply:
It's not really necessary anymore, but I've added a clear as a safeguard